crates/bevy_animation/src/lib.rs

```diff
@@ -1,301 +1,30 @@
-//! Animation for the game engine Bevy
+//! Provides types and plugins for animations.
```

Comment:
If I understand the old code (and assume new one is same), this is not an animation module, it's a `Transform`-only animation module (so skeletal / morph only). This is quite the difference. All the talks I saw about animation where about finding a way to access a generic property to animate, but this is very specific.